src/Directory.Packages.props

```diff
@@ -4,20 +4,20 @@
   </PropertyGroup>
   <ItemGroup>
     <!--    add your PackageVersion nodes here -->
-    <PackageVersion Include="Microsoft.Azure.Kusto.Ingest" Version="12.1.0" />
-    <PackageVersion Include="serilog" Version="2.12.0" />
```

Comment:
AzureDataExplorerSinkOptionsExtensions.cs

Please bump :         private const string ClientVersion = "1.1.0";

Reply:
Just to confirm. Does that not refer to the AppName - Serilog.Sinks.AzureDataExplorer? because I see that the latest version is still at 1.1.0 on nuget. https://www.nuget.org/packages/serilog.sinks.azuredataexplorer

Reply:
We will have to bump the version now since we will have a new package published to Nuget , so has to be reflective of that ?

Reply:
Got it. Thanks!